assignment/impl/src/java/org/sakaiproject/assignment/impl/EmailUtil.java

```diff
@@ -223,6 +217,29 @@ public String htmlContent(AssignmentSubmission submission) {
 
         return buffer.toString();
     }
+    
+    private void submittersContent(AssignmentSubmission submission, boolean isAnon, StringBuilder buffer) {
+		// submitter name and id
+        String submitterNames = "";
+        String submitterIds = "";
+        for (AssignmentSubmissionSubmitter submitter : submission.getSubmitters()) {
```

Comment:
I have changed the previous "for" for a "stream().forEach". I have tried several ways to shorten the method but I don't see how to do it. I have tried using  "Collectors.joining( "," )" to avoid the "if" inside the loop, but it only produces one String as output, so I needed to use two lambda expressions: one for the names and the other for the ids. But this is just the opposite of shortening!

Could you please help me with this?

Thanks for your comments!